pages/common/vue-init.md

```diff
@@ -0,0 +1,17 @@
+# vue init
+
+> Legacy init subcommand for the Vue.js framework.
+> Official templates include `webpack`, `webpack-simple`, `browserify`, `browserify-simple`, and `simple`.
```

Comment:
We can delete this line and include this information in the `vue init` command by writing it as - `{{webpack|webpack-simple..}}`

Reply:
OK, thanks. Making the changes.